plugins/modules/win_domain_computer.py

```diff
@@ -86,6 +86,12 @@
     type: str
     choices: [ absent, present ]
     default: present
+  managed_by:
+    description:
+    - The value to be assigned to the LDAP C(managedBy) attribute.
+    - This value can be in the forms C(Distinguished Name), C(objectGUID),
+      C(objectSid) or C(sAMAccountName), see examples for more details.
+    type: str
```

Comment:
```suggestion
  managed_by:
    description:
    - The value to be assigned to the LDAP C(managedBy) attribute.
    - This value can be in the forms C(Distinguished Name), C(objectGUID),
      C(objectSid) or C(sAMAccountName), see examples for more details.
    type: str
    version_added: '1.3.0'
```
I think you'll need the field for when this option was added. I'm also not certain what the next version of the collection is; current is 1.2.0, so I'm taking a guess at 1.3.0.

Reply:
version_add information added